src/press_quote/quote.macros.html

```diff
@@ -0,0 +1,15 @@
+{#
+  * @overview Macros for press quotes.
+#}
+
+{% from '../icons/icons.macros.html' import svg %}
+
+{#
+  * Renders a single press quote with source logo beneath.
+#}
+{% macro press_quote(content, logo, class=null, id=null) %}
```

Comment:
@josephco I'm adding to cleanup but when we document macros we should be documenting the params in the file, too. Let's treat it the same as JSDoc.